app/utils/is-valid-form.js

```diff
@@ -0,0 +1,27 @@
+import { validateForm } from '@lblod/ember-submission-form-fields';
+
+export function isValidFormFromFormInfo(formInfo) {
```

Comment:
This function name is a bit confusing. Maybe something like `isValidFormInfo` ?

Reply:
Renamed to `isValidForm`